web.rb

```diff
@@ -57,9 +60,84 @@
     "Charge declined"
   end
 
+end 
+
+#endpoint for Stripe Connect Oauth
+get '/connect' do 
+  #gets two params from Stripe OAUTH
+  scope = params[:scope]
+  auth_code = params[:code]
+
+  p scope
+  p auth_code
+
+  #creates account with Stripe or returns error to Sinatra 
+
+  begin
+    response = RestClient.post 'https://connect.stripe.com/oauth/token', {
+      client_secret: $PRIVATE_TEST_KEY,
+      code: auth_code, 
+      grant_type: 'authorization_code'}
+ 
+
+    response = JSON.parse(response.to_s)
+
+    db = SQLite3::Database.new("stripe_store.db")
+    db.execute("INSERT INTO accounts (
+      authorization_code,
+      token_type,
+      stripe_publishable_key,
+      scope,
+      livemode,
+      stripe_user_id,
+      refresh_token,
+      access_token) VALUES (?, ?, ?, ?, ?, ?, ?, ?)", [
```

Comment:
For this many values, it would be typical to set up the value array ahead of time as `rowdata` or something, but that's just to make things less complicated and more readable. Totally optional.